ci/packages/suite-desktop.yml

```diff
@@ -12,6 +12,8 @@
     - yarn install --frozen-lockfile --cache-folder .yarn --prefer-offline
     - yarn build:libs
     - yarn workspace @trezor/suite-data copy-static-files
+    # Get Suite App version from the root package.json
+    - export SUITE_VERSION=$(jq -r .version package.json)
```

Comment:
This won't work as the jq utility is only on deployment runner in nix-shell for this to work I will need to add it back to the base docker image. Also, why do we need to get the suite version here?

Reply:
Nvm, I forgot to get rid of it so it's present in the base docker image. this should work with the current base image sha from develop.

Reply:
The `SUITE_VERSION` variable is used by the yarn task one line below.

https://github.com/trezor/trezor-suite/blob/430a489e24629c0b3944373e9d5c11fae5c20550/packages/suite-desktop/package.json#L18-L20

I agree this is opaque. We technically could pass the `--c.extraMetadata.version` here in the CI manifest like so
```yaml
     - yarn workspace @trezor/suite-desktop build:${platform} --c.extraMetadata.version=$SUITE_VERSION
```

I decided not to because I wanted the `yarn build:xxx` task to fail loudly (eg. when run locally without the `SUITE_VERSION` veriable set) instead of silently using the wrong version from `packages/suite-desktop/package.json`

I added an explaining comment in 430a489e24629c0b3944373e9d5c11fae5c20550 hope that makes it clearer.